ludwig/model_export/coreml_exporter.py

```diff
@@ -0,0 +1,53 @@
+import os
```

Comment:
Please add `from __future__ import annotations` at the very top followed by a blank line.

This will help modernize type hints (and make them easier to add -- without cyclical conflicts).  Thank you!